lib/index.js

```diff
@@ -247,6 +247,20 @@ var parseQuery = function (query, options) {
           var dateEnd = new Date(dateStart.getTime() + 86400000);
           var dateRange = {'$gte': dateStart, '$lt': dateEnd};
           addCondition(lcKey, operator == 'dne' ? { '$not': dateRange }: dateRange);
+        } else if (operator === 'dir') {
+          const dateRangeSeparator = ' to ';
```

Comment:
what `dir` means?  Maybe just `dr` - date range
Space isn't good separator especially in the URL due encodings. Simple coma would work. Range - `start,end`

Reply:
`dir` for date in range, sounds similar to `$in`.

About space, makes sense. Will make the change.

Reply:
@Agrumas , PTAL